integration-test.sh

```diff
@@ -3,9 +3,13 @@ set -e
 
 DIR=$PWD
 
-curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash
-cd ./fabric-samples/first-network
-echo y | ./byfn.sh up
-cp -r crypto-config "$DIR" && cd "$DIR"
+curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash -s -- 2.1.1 1.4.7 0.4.20
+cd ./fabric-samples/test-network
+echo y |  ./network.sh up createChannel
+echo y |  ./network.sh deployCC
+cp -r organizations "$DIR" && cd "$DIR"
+mv organizations crypto-config
```

Comment:
why don't we rename `crypto-config` in our own code to `organizations`?

Reply:
k